net/kcat/Portfile

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        edenhill kcat 1.7.1
+categories          net
+platforms           darwin
+license             BSD
+maintainers         {gmail.com:slashapp @slashapp} openmaintainer
```

Comment:
You are de facto changing the maintainer on an existing port here. Does @alexeyt820 feel okay with that?

Reply:
You aren't "adding" a maintainer. You're changing the maintainer of an old port. Effectively this is just a renaming, but you're substituting yourself for @alexeyt820, the maintainer up until now. Is that person okay with this?

Reply:
This port hasn't been updated in 4 years, I just wanted to update the version, nothing else.

Reply:
Then you didn't need to add yourself as the maintainer. In any case, if the timeout expires we'll assume @alexeyt820 is okay with it.

Reply:
I'm ok with it